browser/search_engines/search_engine_tracker.cc

```diff
@@ -52,6 +52,8 @@ SearchEngineP3A GetSearchEngineProvider(const GURL& search_engine_url,
     result = SearchEngineP3A::kDaum;
   } else if (type == SEARCH_ENGINE_NAVER) {
     result = SearchEngineP3A::kNaver;
+  } else if (type == SEARCH_ENGINE_YAHOO) {
```

Comment:
Q) It seems `SEARCH_ENGINE_YAHOO` type is used by many regional yahoo(yahoo_it, yahoo_hk and etc..).
So, I think it's difficult to know this is from JP by using `SEARCH_ENGINE_YAHOO` type only.
Maybe we'll filtering in p3a backend?

Reply:
added a `.jp` host suffix check

Reply:
How about comparing more precisely like this?
```
search_engine_url.host_piece() == GURL(TemplateURLPrepopulateData::brave_yahoo_jp.search_url).host_piece()
```